app/presenters/nav_presenter.rb

```diff
@@ -34,28 +34,30 @@ def navs
         subnavs: [
           { label: I18n.t('nav.tabs.admin_dashboard'),
             path: %i[admin dashboard],
-            controllers: ['dashboard'],
-            actions: ['admin'] },
+            controllers: ['admin/dashboard'] },
           { label: I18n.t('nav.tabs.admin_users'),
             path: %i[admin users],
-            controllers: ['admin/users'],
-            actions: %w[index new edit] },
+            controllers: ['admin/users'] },
           { label: I18n.t('nav.tabs.admin_group_types'),
             path: %i[admin group_types],
-            controllers: ['admin/group_types', 'admin/user_groups'],
-            actions: %w[index new edit] },
-          { label: I18n.t('nav.tabs.time_ranges'),
-            path: %i[admin time_ranges],
-            controllers: ['admin/time_ranges'],
-            actions: %w[index new edit] }
```

Comment:
time_ranges removed by accident? This is the Data tab on the user view.

Reply:
yes accidental merge removal! adding back in